<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpriteGenerator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images/images/spritegenerator.svg</normaloff>:/images/images/spritegenerator.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCommandLinkButton" name="createSpriteCommandButton">
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save css sprite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="previewPageCommandButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Create preview page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="imageNameHorizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Element margin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="marginSpinBox">
             <property name="suffix">
              <string>px</string>
             </property>
             <property name="maximum">
              <number>80</number>
             </property>
             <property name="value">
              <number>8</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Element information</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="imageNameHorizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Image name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fileName">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="imageSizeHorizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Image size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="imageSizeX">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="imageSizeY">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>314</width>
              <height>234</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="elementImageLabel">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Generated css values</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="resultingCssTextBrowser">
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddFile"/>
    <addaction name="actionAddDirectory"/>
    <addaction name="actionRemoveFile"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFile"/>
   <addaction name="actionAddDirectory"/>
   <addaction name="actionRemoveFile"/>
  </widget>
  <action name="actionAddFile">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/images/22x22/archive-insert.png</normaloff>:/images/images/22x22/archive-insert.png</iconset>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
   <property name="toolTip">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionAddDirectory">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/images/22x22/archive-insert-directory.png</normaloff>:/images/images/22x22/archive-insert-directory.png</iconset>
   </property>
   <property name="text">
    <string>Add Directory</string>
   </property>
   <property name="toolTip">
    <string>Add Directory</string>
   </property>
  </action>
  <action name="actionRemoveFile">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/images/22x22/archive-remove.png</normaloff>:/images/images/22x22/archive-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove File</string>
   </property>
   <property name="toolTip">
    <string>Remove File</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
